### PR TITLE
Semi-automatic update of generated content

### PR DESCRIPTION
**Pull request by taylorbot**
This PR has been created automatically by the 'Update generated content' Github workflow
to keep

- Changes and releases
- CLI versions
- Cluster Apps
- CRD pages
- Website header and footer

up-to-date.

This PR is opened or updated every weekday morning. On Fridays, it is merged by 'merge-autogenerated-content' workflow. In case of being triggered manually you need to merge it manually too in order to have the content published.

### CODE REVIEW

File: src/content/changes/managed-apps/kong-app/v4.0.0.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ title: kong-app release v4.0.0
 This is a major release because it changes the default behaviour of the Chart.
 - For Kong Enterprise deployment the flag `.Values.image.repository` and `.Values.image.tag` can now be omitted.
 - For Kong OSS deployment the flag `.Values.image.repository` and `.Values.image.tag` now have to be set to the respective OSS values.
-For further instructions please refer to the [README.md](https://github.com/giantswarm/kong-app/blob/main/README.md).
+For further instructions please refer to the [README.md](./README.md).
```
Comment (AndiDog) [inline src/content/changes/managed-apps/kong-app/v4.0.0.md:20]: @pipo02mix At best we would fix relative links when copying these Markdown files

Comment (marians) [reply src/content/changes/managed-apps/kong-app/v4.0.0.md:20]: We definitely can't fix it here manually 🤷

Comment (pipo02mix) [reply src/content/changes/managed-apps/kong-app/v4.0.0.md:20]: you mean to fix it in the automation? it is a good idea, yeah

Comment (pipo02mix) [reply src/content/changes/managed-apps/kong-app/v4.0.0.md:20]: I will address this in incoming PR

**Pull request merged**